src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,106 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static int DEFAULT_CAPACITY = 16;
+    private static double LOAD_FACTOR = 0.75;
+    private static int CONSTTANT = 2;
```

Comment:
Constant should have self descriptive and informative naming, like you did this before for LOAD_FACTOR constant.
```suggestion
    private static int CONSTTANT = 2;
```